compatibility_server/test_pip_checker.py

```diff
@@ -95,8 +94,8 @@ def exec_run(self, cmd, stdout=True, stderr=True):
         duration = current_time - self.start_time
 
         if duration > pip_checker.TIME_OUT:
-            raise docker.errors.APIError(message="time out",
-                                         explanation="Request time out.")
```

Comment:
This wasn't mocking the docker container timeout behavior correctly before, which should actually return a 137 code instead of raising a `docker.errors.APIError`. Changing this to match the behavior and the test in line 123 is for testing the timeout error.